src/Nethermind/Nethermind.Consensus.Clique/ICliqueModule.cs

```diff
@@ -24,15 +24,34 @@ namespace Nethermind.Consensus.Clique
     [RpcModule(ModuleType.Clique)]
     public interface ICliqueModule : IModule
     {
+        [JsonRpcMethod(Description = "Retrieves a snapshot of all clique state at a given block.", Returns = "Object", IsImplemented = true)]
```

Comment:
Retrieves a clique snapshot for a given block containing information about votes cast and a list of validators.